data/mods/ssb/abilities.ts

```diff
@@ -1323,6 +1323,20 @@ export const Abilities: {[k: string]: ModdedAbilityData} = {
 		gen: 8,
 	},
 
+	// nui
+	conditionoverride: {
+		desc: "On switch-in, this Pokemon attracts the opponents regardless of gender. Pokemon who are attracted have their Special Defense stat reduced by 25%.",
+		shortDesc: "Attracts opponent. Attracted Pokemon have SpD reduced by 25%.",
+		onStart(pokemon) {
+			const target = pokemon.side.foe.active[0];
+			target.addVolatile('attract');
+			this.add('-start', target, 'Attract', '[from] ability: Condition Override', '[of] ' + pokemon);
+		},
```

Comment:
nO

This should just give jiggly the ability to. Not auto do it on switch in pls